server/seeders/20230622202911-addAtBrowser.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.createTable('AtBrowsers', {
+            atId: { type: Sequelize.DataTypes.INTEGER },
+            browserId: { type: Sequelize.DataTypes.INTEGER }
+        });
```

Comment:
This seems out of place in a seeder. As this is about database structure, I'd think this `createTable` block belongs in its own migration file instead.